src/glyph/Glyph.js

```diff
@@ -64,23 +64,12 @@ export default class Glyph {
 
     let {advance:advanceWidth, bearing:leftBearing} = this._getTableMetrics(this._font.hmtx);
 
-    // For vertical metrics, use vmtx if available, or fall back to global data from OS/2 or hhea
+    // For vertical metrics, use vmtx if available, or fall back to global data
     if (this._font.vmtx) {
       var {advance:advanceHeight, bearing:topBearing} = this._getTableMetrics(this._font.vmtx);
-
     } else {
-      let os2;
-      if (typeof cbox === 'undefined' || cbox === null) { ({ cbox } = this); }
```

Comment:
This line is important @StLyn4. `cbox` is only passed when called by `_getPhantomPoints`.

Reply:
Makes sense. Thanks for the clarification. The only problem is that the library seems to be dead and I'm not sure if it even makes sense to fix :(

Reply:
I think it makes sense, since others may use the PRs on their own forks as needed, as we did.